### PR TITLE
chore(theme-translations): complete ko translations

### PR DESCRIPTION
**Pull request by HyunseungLee-Travis**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).

## Motivation

This PR translate some english texts still untranslated.
It also fixed some unnatural wordings.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
cc @revi @koko8829 @winterlood Could you review this please?

File: packages/docusaurus-theme-translations/locales/ko/plugin-ideal-image.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-  "theme.IdealImageMessage.404error": "404. Image not found",
-  "theme.IdealImageMessage.error": "Error. Click to reload",
-  "theme.IdealImageMessage.load": "Click to load{sizeMessage}",
-  "theme.IdealImageMessage.loading": "Loading...",
-  "theme.IdealImageMessage.offline": "Your browser is offline. Image not loaded"
+  "theme.IdealImageMessage.404error": "404. 이미지를 찾을 수 없습니다",
+  "theme.IdealImageMessage.error": "Error. 클릭해서 새로고침하기",
+  "theme.IdealImageMessage.load": "클릭해 {sizeMessage} 로드",
+  "theme.IdealImageMessage.loading": "로딩중...",
+  "theme.IdealImageMessage.offline": "오프라인 상태. 이미지 로드하지 않았습니다"
```
Comment (revi) [inline packages/docusaurus-theme-translations/locales/ko/plugin-ideal-image.json:6]: 자연스러운 문장이 되도록 조사를 넣어 주세요.

File: packages/docusaurus-theme-translations/locales/ko/theme-common.json [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
   "theme.NotFound.p1": "원하는 페이지를 찾을 수 없습니다.",
   "theme.NotFound.p2": "사이트 관리자에게 링크가 깨진 것을 알려주세요.",
   "theme.NotFound.title": "페이지를 찾을 수 없습니다.",
   "theme.TOCCollapsible.toggleButtonLabel": "이 페이지에서",
-  "theme.admonition.caution": "caution",
-  "theme.admonition.danger": "danger",
-  "theme.admonition.info": "info",
-  "theme.admonition.note": "note",
-  "theme.admonition.tip": "tip",
+  "theme.admonition.caution": "조심",
```
Comment (revi) [inline packages/docusaurus-theme-translations/locales/ko/theme-common.json:15]: 조심 보다는 주의 가 나을 것 같네요.

**Review by revi — COMMENTED** [review]
(Was about to select `Request changes` but I seemingly lack the permission to do so.)

로딩 중 이라는 단어에 대해서는… 로딩 이라는 단어 자체가 load+ing인데 ~ing 자체가 ~중 이므로 로딩중이면 로드중중 이 되어서 비문이 되지 않나..? 싶기도 하고 관용적어구로 넘어가도 되나 싶고 애매하네요.

**Comment by Josh-Cena** [comment]
> (Was about to select `Request changes` but I seemingly lack the permission to do so.)

Yeah, that's a bit annoying :/

Thanks for the review!

**Comment by Josh-Cena** [comment]
We can merge this if any of the reviewers can give a thumbs-up.

**Review by Josh-Cena — APPROVED** [review]
All reviews seem cleared now—if any of you have further ideas of improvements feel free to follow up.

@HyunseungLee-Travis Thanks for your contribution!

**Pull request merged**